src/JBRulesets.sol

```diff
@@ -388,17 +388,17 @@ contract JBRulesets is JBControlled, IJBRulesets {
             revert INVALID_DECAY_RATE();
         }
 
-        // Weight must fit into a uint88.
-        if (weight > type(uint88).max) revert INVALID_WEIGHT();
+        // Weight must fit into a uint112.
+        if (weight > type(uint112).max) revert INVALID_WEIGHT();
```

Comment:
Should we make an `JBConstants.MAX_WEIGHT`?

Reply:
meh. not really useful for any division calculations